engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/hydration/NadelHydrationInputBuilder.kt

```diff
@@ -140,7 +146,7 @@ internal class NadelHydrationInputBuilder private constructor(
         )
     }
 
-    private fun getResultValue(
+    private fun getSingleResultValue(
```

Comment:
renamed because if you look at it - its does `emptyOrSingle` on the list of values